sci-libs/shapelib/files/0013-Fix-cppcheck-nullPointerOutOfMemory-warnings.patch

```diff
@@ -0,0 +1,731 @@
+From 2973fa0b5253514d709ef8f189e90ccc2c58602a Mon Sep 17 00:00:00 2001
```

Comment:
While this is a correctness fix, it's _very_ minor and a pretty large patch. I don't think we need it.